modules/patched/CODEREVIEW

```diff
@@ -0,0 +1,3 @@
+// @CODEREVIEW: What is going on in this folder?
+// @CODEREVIEW: It's not obvious what is being patched and from which sources.
+// @CODEREVIEW: Consider using a make file in the profile to apply patches.
```

Comment:
This is a placeholder for keeping all modules that have been patched.
It's true that the sources of patches is not very obvious. We could add a README file listing that (luckily there are not too many modules here).

Regarding the suggestion about adding a MAKE file, as I understand it would would require to move ALL the contrib modules to that MAKE file instead of having them in the repo as plain files.
Since it would drastically change the structure of the repo I suggest having everybody agreed before performing any changes.

cc @skifter

Reply:
cc @rimi-itk

Reply:
@stanbellcom Please add this README file with notes on the source of the code and the source of the patches (if external). And maybe a few comments on why the patch is applied.